connectors/kubernetes-connector/main.go

```diff
@@ -59,16 +60,37 @@ func configHandler(data []byte) {
 	tMonConn := &transit.MonitorConnection{Extensions: tExt}
 	tMetProf := &transit.MetricsProfile{}
 
-	if err := connectors.UnmarshalConfig(data, tMetProf, tMonConn); err != nil {
+	err := connectors.UnmarshalConfig(data, tMetProf, tMonConn)
+	if err != nil {
 		log.Err(err).Msg("Could not parse config")
 		return
 	}
 
-	if tMonConn.Extensions.(*ExtConfig).AuthType == ConfigFile {
-		if err := writeDataToFile([]byte(tMonConn.Extensions.(*ExtConfig).KubernetesConfigFile)); err != nil {
-			log.Err(err).Msg("Could not write to file")
+	var yamlData []byte
+	switch tMonConn.Extensions.(*ExtConfig).AuthType {
+	case ConfigFile:
+		yamlData = []byte(tMonConn.Extensions.(*ExtConfig).KubernetesConfigFile)
+	default:
+		yamlData, err = yaml.Marshal(struct {
+			Auth                   AuthType `yaml:"auth"`
+			EndPoint               string   `yaml:"host"`
+			KubernetesBearerToken  string   `yaml:"token,omitempty"`
```

Comment:
This should be at the end to match order of data.

Reply:
Fixed